### ISSUE TITLE
Can't compile: #error "Unsupported Compiler!"  /* Compiler defines not found */

### ISSUE THREAD

**Issue by mtbspace** [platformio/platform-ststm8#22]
I'm trying to compile a new project.  I've not added any additional code to main.cpp

My platformio.ini file is as follows:
```
[env:stm8sblack]
platform = ststm8
board = stm8sblack
framework = arduino
```
I'm using Visual Studio Code on debian buster
The compiler output is:
```
Compiling .pio/build/stm8sblack/src/main.cpp.rel
In file included from /home/james/.platformio/packages/framework-arduinoststm8/cores/sduino/Arduino.h:32,
                 from src/main.cpp:1:
/home/james/.platformio/packages/framework-arduinoststm8/STM8S_StdPeriph_Driver/inc/stm8s.h:90:3: error: #error "Unsupported Compiler!"
  #error "Unsupported Compiler!"          /* Compiler defines not found */
```

**Comment by mtbspace** [comment]
I've tried to make it use SDCC.
I've also tried renaming main.cpp to main.c as others have suggested (and I'm not sure how I could have ended up with the wrong file extension), 
Neither of these actions have helped.

**Comment by mtbspace** [comment]
Closed because it's a duplicate of #21
